### PR TITLE
make 2xx response status code compatible with docker engine

### PR DESCRIPTION
**Pull request by allencloud**
make 2xx response status code compatible with docker
1. POST /images/load    from 201 to 200
2. POST /images/{name:.*}/tag from 200 to 201
3. POST /containers/{name:.*}/rename from 200 to 204

Signed-off-by: Sun Hongliang <allen.sun@daocloud.io>

### CODE REVIEW

File: api/handlers.go [hunk 1]
```diff
@@ -1273,6 +1275,8 @@ func postRenameContainer(c *context, w http.ResponseWriter, r *http.Request) {
 			httpError(w, err.Error(), http.StatusInternalServerError)
 		}
 	}
+	w.Header().Set("Content-Type", "application/json")
```
Comment (vieux) [inline api/handlers.go]: I don't see why we need to put a content-type since there is no content.

**Comment by allencloud** [comment]
Thanks @vieux  I did some change.

**Comment by vieux** [comment]
@allencloud my comment in still here: `w.WriteHeader(http.StatusCreated)` doesn't need a type.

**Comment by allencloud** [comment]
@vieux  Yes, of course. 
I did the change.

**Comment by vieux** [comment]
LGTM ping @dongluochen @nishanttotla @abronan

**Comment by dongluochen** [comment]
LGTM

**Pull request merged**